proposed/clock-meta.md

```diff
@@ -0,0 +1,154 @@
+# PHPDoc Meta Document
```

Comment:
This article is not for [PHPDoc](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc-meta.md).
Maybe replaced by "Clock Meta Document", like [Container Meta Document](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-11-container-meta.md).